src/scss/components/navigation/_table-of-contents.scss

```diff
@@ -0,0 +1,85 @@
+.table-of-contents {
+  @include components-container-max-width();
+  margin-left: 0;
+  margin-right: 0;
+
+  @include breakpoint($breakpoint-m) {
+    @include component-side-padding();
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .has-sidebar & {
+    @include component-negative-side-margin;
+  }
+
+  // In rare cases TPR Service page can be rendered without sidebar.
+  // In these cases remove the left and right paddings from table of contents.
+  .service-header__container:not(.has-sidebar) & {
+    @include breakpoint($breakpoint-m) {
+      padding-left: 0;
+      padding-right: 0;
+    }
+  }
+}
+
+.table-of-contents__container {
+  @include component-margin-top();
+  background-color: $hdbt-color-white;
+  border-left: 8px solid var(--hdbt-color-gold--primary);
+  max-width: $content-width-max;
+  padding: $spacing-double $spacing-and-half;
+}
+
+.table-of-contents__title {
+  @include font-size(24px, 28px);
+  margin-bottom: 0;
+  margin-top: 0;
+
+  @include breakpoint($breakpoint-l) {
+    @include font-size(36px, 42px);
+  }
+}
+
+.table-of-contents__list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.table-of-contents__item {
+  padding-top: $spacing;
+
+  @include breakpoint($breakpoint-l) {
+    padding-top: $spacing-and-half;
+  }
+}
+
+.table-of-contents__link {
+  @include font-size(20px, 28px);
+  display: inline-block;
+  font-weight: $font-weight-medium;
+  position: relative;
+  text-decoration: underline;
+
+  @include breakpoint($breakpoint-l) {
+    @include font-size(20px, 28px);
+  }
+
+  &:focus {
```

Comment:
If this focus style is a new standard, should we maybe create a mixin for it instead of building it multiple times in the code?

Reply:
True! But in this case I just wanted to add some kind of focus and ended up basically copying it from the navigation links. I think we should wait until the "hovers and focuses" story has been completed and we _really know_ how each different type of link, button, whatnot hover/focus/active should work/look like. What do you think?

Reply:
Sure, it's just hunting all these places can be a bore. Maybe a //TODO?

Reply:
Added TODO. 👍